src/tests/stdlib/create_symbolic_link.ab

```diff
@@ -7,5 +7,5 @@ main {
     } else {
         echo "failed"
     }
-    trust $rm {tmpdir}$
+    trust $rm -fr {tmpdir}$
```

Comment:
I believe this is also part of a different pending PR.  Shouldn't cause merge conflicts, but this is implemented in a commit of its own, so I can drop it and rebase if it causes problems.